src/material-experimental/mdc-slide-toggle/slide-toggle.ts

```diff
@@ -176,7 +188,7 @@ export class MatSlideToggle implements ControlValueAccessor, AfterViewInit, OnDe
       this._foundation.setDisabled(this._disabled);
     }
   }
-  private _disabled = false;
+  _disabled = false;
```

Comment:
This doesn't need to be exposed either, it can be set through `disabled`.

Reply:
@crisbeto Whoops, just unexposed them.